File: Python/Find-the-idx-of-2nd-occurrence-of-letter.py
def second_symbol(s, symbol):
    count = 0

    for idx, letter in enumerate(s):
        if letter == symbol:
            count += 1

            if count == 2:
                return idx

    return -1

# second_symbol('Hello world!!!','l') --> 3
# second_symbol('Hello world!!!', 'A') --> -1

File: JavaScript/Find-the-idx-of-2nd-occurrence-of-letter.js
function secondSymbol(s, symbol) {
  const indices = [...s].reduce((acc, element, index) => {
    if (element === symbol) acc.push(index);
    return acc;
  }, []);

  return indices[1] ?? -1;
}

// second_symbol('Hello world!!!','l') --> 3
// second_symbol('Hello world!!!', 'A') --> -1
